sros2/test/sros2/commands/security/verbs/test_create_keystore.py

```diff
@@ -31,24 +31,41 @@ def keystore_dir(tmpdir_factory):
     keystore_dir = str(tmpdir_factory.mktemp('keystore'))
 
     # Create the keystore
-    assert cli.main(argv=['security', 'create_keystore', keystore_dir]) == 0
+    assert cli.main(argv=['security', 'create_keystore', '-k', keystore_dir]) == 0
 
     # Return path to keystore directory
-    return keystore_dir
+    return Path(keystore_dir)
 
 
 def test_create_keystore(keystore_dir):
-    expected_files = (
-        'ca.cert.pem', 'ca.key.pem', 'governance.p7s', 'governance.xml'
+    public = keystore_dir / 'public'
+    private = keystore_dir / 'private'
+    contexts = keystore_dir / 'contexts'
+    expected_files_public = (
+        public / 'ca.cert.pem',
+        public / 'permissions_ca.cert.pem',
+        public / 'identity_ca.cert.pem',
+    )
+    expected_files_private = (
+        private / 'ca.key.pem',
+        private / 'permissions_ca.key.pem',
+        private / 'identity_ca.key.pem',
+    )
+    expected_files_contexts = (
+        contexts / 'governance.p7s',
+        contexts / 'governance.xml',
     )
-    assert len(os.listdir(keystore_dir)) == len(expected_files)
 
-    for expected_file in expected_files:
-        assert os.path.isfile(os.path.join(keystore_dir, expected_file))
+    assert len(list(keystore_dir.iterdir())) == 3
+    assert len(list(public.iterdir())) == len(expected_files_public)
+    assert len(list(private.iterdir())) == len(expected_files_private)
+    assert len(list(contexts.iterdir())) == len(expected_files_contexts)
+    expected_files = expected_files_public + expected_files_private + expected_files_contexts
+    assert all(x.exists() for x in expected_files)
```

Comment:
Nit: use `isfile` instead of `exists`

Reply:
Done, see https://github.com/ros2/sros2/pull/177/commits/2db3d4e8bff1b56e6621b170a78feeafcd4b1fbd.